package.json

```diff
@@ -104,6 +104,7 @@
   "devDependencies": {
     "broccoli-asset-rev": "^2.4.2",
     "ember-cli": "~2.8.0",
+    "ember-cli-app-version": "^2.0.0",
```

Comment:
i think we need to take this out. we determined in the previous PR that it is unused.